### PR TITLE
Fix #4257 - Install the Energy+.schema.epJSON

### LINKED ISSUE #4257: OS v3.2 installer missing Energy+.schema.epJSON
Issue overview
--------------
I was trying to run a simple script to test epJSON capabilities in the upcoming OS:
```
require 'openstudio'
idfPath = OpenStudio::Path.new(File.join(File.dirname(__FILE__), "in.idf"))
idfFile = OpenStudio::IdfFile.load(idfPath,"EnergyPlus".to_IddFileType).get
puts OpenStudio::EPJSON::toJSONString(idfFile);
```
And it returned
```
[epJSONTranslator] <1> Schema is invalid at path=/home/shorowit/OpenStudio-3.1.1-alpha+6c050c1e03-Ubuntu-18.04/usr/local/openstudio-3.1.1-alpha/EnergyPlus/Energy+.schema.epJSON
null
```

Indeed usr/local/openstudio-3.1.1-alpha/EnergyPlus/Energy+.schema.epJSON does not exist. Just missing from the installer?

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------

Fix #4257 - Install the Energy+.schema.epJSON

### Pull Request Author

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.

 - [ ] Model API Changes / Additions
 - [ ] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [ ] Model API methods are tested (in `src/model/test`)
 - [ ] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [ ] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Checked behavior in OpenStudioApplication, adjusted policies as needed (`src/openstudio_lib/library/OpenStudioPolicy.xml`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [ ] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [ ] If change to an IDD file, add the label `IDDChange`
 - [ ] If breaking existing API, add the label `APIChange`
 - [ ] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1250,6 +1250,9 @@ file(GLOB ENERGYPLUS_LIB_FILES "${ENERGYPLUS_DIR}/libenergyplus*")
 file(GLOB EXPAND_OBJECTS "${ENERGYPLUS_DIR}/ExpandObjects*")
 
 install(FILES "${ENERGYPLUS_IDD}" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
+# Get epJSON schema too
+string(REPLACE ".idd" ".schema.epJSON" ENERGYPLUS_EPJSON_SCHEMA ${ENERGYPLUS_IDD})
+install(FILES "${ENERGYPLUS_EPJSON_SCHEMA}" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
```
Comment (jmarrec) [inline CMakeLists.txt:1255]: Install epsjon schema

File: CMakeLists.txt [hunk 2]
```diff
@@ -1263,7 +1266,7 @@ if(APPLE)
   install(PROGRAMS "${ENERGYPLUS_DIR}/libintl.8.dylib" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
   #install(PROGRAMS "${ENERGYPLUS_DIR}/libpythonwrapper.dylib" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
   install(PROGRAMS "${ENERGYPLUS_DIR}/libgcc_s.1.dylib" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
-  install(PROGRAMS "${ENERGYPLUS_DIR}/ssc.dylib" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus)
+  install(PROGRAMS "${ENERGYPLUS_DIR}/ssc.dylib" DESTINATION ./EnergyPlus/ COMPONENT EnergyPlus) # TODO: Next E+ installer we use will have ssc static, so remove ssc.dylib/so/dll
```
Comment (jmarrec) [inline CMakeLists.txt:1269]: Note to remember to scrap that on all three platforms when we update the E+ installer.

**Comment by shorowit** [comment]
I can confirm that this allowed me to generate the epJSON via the SDK methods. Thanks @jmarrec!

(I wasn't able to run the simulation though, but it seems to be an [EnergyPlus issue](https://github.com/NREL/EnergyPlus/issues/8654).)

**Review by shorowit — APPROVED** [review]

**Comment by jmarrec** [comment]
Thanks for testing!

**Review by tijcolem — APPROVED** [review]

**Pull request merged**